src/backend/settings.py

```diff
@@ -522,3 +522,12 @@
 
 #: Whether thumbnails should be stored in high resolution (used by :doc:`easy-thumbnails:index`)
 THUMBNAIL_HIGH_RESOLUTION = True
+
+##############
+# Pagination #
+##############
+
+#: Number of entries displayed per pagination chunk
+#: see :attr:`django-pagination:django.core.paginator.Paginator`
+
+PER_PAGE = 16
```

Comment:
```suggestion
#: Number of entries displayed per pagination chunk
#: see :class:`~django.core.paginator.Paginator`
PER_PAGE = 16
```
Oh, and yet another thing: If you use the syntax `#:` for docstrings, don't include a newline between the docstring in the comment, otherwise it will be ignored and the setting won't be contained in the documentation.